src/Jobs/DeleteTenantStorage.php

```
@@ -22,17 +34,11 @@ public function __construct(
 
     public function handle(): void
     {
-        if (config('tenancy.filesystem.suffix_storage_path') === false) {
-            // Skip storage deletion if path suffixing is disabled
-            return;
-        }
-
-        $centralStoragePath = tenancy()->central(fn () => storage_path());
-        $tenantStoragePath = tenancy()->run($this->tenant, fn () => storage_path());
+        $tenantStoragePath = FilesystemTenancyBootstrapper::getBoundTenantStoragePath($this->tenant);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Stancl\Tenancy\Bootstrappers\FilesystemTenancyBootstrapper' in method 'handle'. (undefined)

(StaticAccess)